config/database.yml

```diff
@@ -0,0 +1,15 @@
+development:
```

Comment:
Normally this file is in `.gitignore`, and the example is for you to build it according to that. This may require a username though and that is not a public detail. You can overcome it with `<%= ENV["USER"] %>`. You can use ERB in yaml files here.